### DISCUSSION TITLE
Call SIP Search in Homer 10

### DISCUSSION THREAD

**Discussion by zppinto** [sipcapture/homer#661] [Q&A] [answered]
Where can I find this interface in the Homer 10?
As far I understood we only have Grafana available now. Is there any way to have the same interface there?

![image](https://github.com/user-attachments/assets/4824e21f-c3b3-4ad5-9d65-c82f4e6b5208)

**Comment by lmangani** [comment, marked as answer]
You will not. If you want this interface, stick to HOMER 7.x for the moment

**Reply by zppinto** [reply]
Thank you for your reply.
Are there any plans to make it available in the future?

**Comment by lmangani** [comment]
Well, It is already available in the present and it is maintained :) 
HOMER 10 is a design preview and once completed a new UI will most likely be developed
